### DISCUSSION TITLE
Can you require text responses using the survey text plugin?

### DISCUSSION THREAD

**Discussion by SaoirseConnorDesai** [jspsych/jsPsych#729] [Q&A] [answered]
Is there a way to require people to enter a text response using the survey text plugin?

**Comment by vijaymarupudi** [comment, marked as answer]
Does https://groups.google.com/forum/#!topic/jspsych/9svnvEFIl88 solve your issue?

**Reply by jodeleeuw** [reply]
Copying that answer here to make this answer a bit easier to find.

> The objects in the questions parameter accept a required key/value pair.
> 
> ```js
> {
>   "type": "survey-text",
>   "questions": [
>     {
>       "prompt": "What is your name?",
>       "placeholder": "Name",
>       "required": true,
>       "name": "participantName"
>     }
>   ]
> }
> ```
